src/utils/env.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @flow
+ * @file File for some simple environment-related utilities
+ * @author Box
+ */
+
+/**
+ * Return true if we are currently running in a test or development
+ * environment.
+ *
+ * @return {boolean} true if we are running in a test or dev environment
+ */
+export default function isDevEnvironment() {
+    return process.env.NODE_ENV === 'test' || process.env.NODE_ENV === 'dev';
```

Comment:
What happens if `process` is undefined, such as in a browser? Have the components been tested in a browser?

Reply:
https://webpack.js.org/guides/production/#specify-the-mode

I have tested it in a browser, but in a mini React app I created in an ad-hoc, only-on-my-machine sort of way. I should think about doing it more formally perhaps.

Also, it runs in the browser in the style guide.

Reply:
We have a babel-only build as well that outputs to the `/es` folder. It seems like this could break for customers who aren't using webpack as their build system.